script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -231,29 +233,31 @@ But even so, there are some things on which he cannot compromise.
 [sha:538dfd4a717831cd6ff970210cf4e4fabedb9bd9]{
 -- Page 672, Offset 10463.
 -- それでは<手|・><ぬ|・><る|・><い|・>と読みながら、長槍に残された力の五割をたたきこむ。
-Thinking that it might be too late,
- he throws half of his remaining strength into his lance.
+He throws half of his remaining strength into the lance,
+ even as he's fully convinced that doing so will prove insufficient.
```

Comment:
Yeah, "fully convinced" maybe overreaching.
That said, this goes up to the lines where he is talking about how he _knows_ should destroy all of it, but because of how much fun he's having, he holds back here. So he is fairly certain, and "fears" read as him being a bit too unconfident in his prediction.
Maybe just drop 'fully'? It's a good collocation, but might might him sound overly certain.

Reply:
I guess yeah,
> even though he knows that [doing so/that amount] will prove insufficient.

is fine. I hadn't been reading it as him holding back because he's having too much fun but holding back to keep some in the tank.

Reply:
It's that and he doesn't wanna use his wifeys souls.